client/src/components/exercise/lab4/pages/FormSkipToMainBroken.js

```diff
@@ -31,7 +31,7 @@ class FormSkipToMainBroken extends Component {
             <Fragment>
                 <ExtraNav />
                 <AppInstructions
-                    class={this.state.class}
+                    className={this.state.class}
```

Comment:
Do you want to maybe destructure your class state:
const { class } = this.state

That way you can just pass it in as class instead of typing this.state.class over and over again